test/set.js

```diff
@@ -6,7 +6,6 @@ var oldConfigSilent = shell.config.silent;
 shell.config.silent = true;
 
 shell.rm('-rf', 'tmp');
-shell.mkdir('tmp');
```

Comment:
One of the tests assumed incorrect semantics of `cp()`. When I fixed the semantics, I realized this test was failing.

The directory actually gets created with a `cp()` command on [this line](https://github.com/shelljs/shelljs/blob/fix-cp-r-option/test/set.js#L20)